system-parachains/asset-hubs/asset-hub-kusama/src/lib.rs

```diff
@@ -1029,6 +1030,38 @@ impl pallet_remote_proxy::Config for Runtime {
 	type WeightInfo = weights::pallet_remote_proxy::WeightInfo<Runtime>;
 }
 
+parameter_types! {
+	pub const DepositPerItem: Balance = system_para_deposit(1, 0);
+	pub const DepositPerByte: Balance = system_para_deposit(0, 1);
```

Comment:
Hope this implies a standard eg. ERC20 contract will be dirt cheap :)

Reply:
this translates to 1 MILLICENTS per byte